continuous_integration/environment-mindeps.yaml

```diff
@@ -0,0 +1,35 @@
+name: dask-distributed
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - click=6.7
+  - cloudpickle=1.5.0
+  - jinja2
+  - msgpack-python=0.6.0
+  # There are known issues when running Python and openssl 3 (see
+  # https://bugs.python.org/issue38820).
+  # Openssl 1.1.1 contains a backport to fix unexpected SSL errors being raised
+  # https://github.com/openssl/openssl/pull/11400.
+  # Make sure openssl < 3 is installed as a temporary workaround.
+  - openssl<3
+  - psutil=5.4.7
+  - sortedcontainers=2.0.4
+  - tblib=1.6.0
+  - toolz=0.10.0
+  - tornado=5
+  - zict=0.1.3
+  - pyyaml
+  - setuptools
+  # test dependencies
+  - pytest
+  - pytest-asyncio<0.14.0
+  - pytest-faulthandler
+  - pytest-repeat
+  - pytest-rerunfailures
+  - pytest-timeout
+  - pip
+  - pip:
+    # Distributed depends on the latest version of Dask
+    - git+https://github.com/dask/dask
```

Comment:
We occasionally have a lower version of dask than the version of distributed in requirements.txt.
Shouldn't we take the opportunity to test this by pinning dask=2021.12.0?